CMakeLists.txt

```diff
@@ -41,14 +55,44 @@ include(BuildSettings)
 include(Packages)
 include(ClangFormat)
 
-################################################################################
-#   tomopy source
-################################################################################
+# ##############################################################################
+# tomopy source
+# ##############################################################################
 
 add_subdirectory(source)
 
-################################################################################
-#   Information
-################################################################################
+# ##############################################################################
+# Information
+# ##############################################################################
 
 print_features()
+
+install(EXPORT libtomoTargets
+        FILE libtomoTargets.cmake
+        NAMESPACE libtomo::
+        DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/libtomo
+)
+
+include(CMakePackageConfigHelpers)
+
+write_basic_package_version_file(
+  ${CMAKE_CURRENT_BINARY_DIR}/libtomoConfigVersion.cmake
+  VERSION ${PACKAGE_VERSION}
+  COMPATIBILITY SameMajorVersion)
+
+configure_package_config_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/Config.cmake.in
+  "${CMAKE_CURRENT_BINARY_DIR}/libtomoConfig.cmake"
+  INSTALL_DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/libtomo
```

Comment:
I personally like to use `${CMAKE_INSTALL_DATAROOTDIR}/libtomo/cmake` which usually translates to `share/libtomo/cmake` because `${CMAKE_INSTALL_LIBDIR}` is `lib` on some systems and `lib64` on others.

Reply:
Thanks, but I am going to follow the the [notes in the Kitware CMake wiki](https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/Packaging).

Reply:
They just use `${CMAKE_INSTALL_LIBDIR}/cmake/libtomo` as an example. You can use any of the [paths listed in the find_package docs](https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-search-procedure). As you can see, both paths are actually technically for Unix `(U)` so you might want to adopt one that conforms to both Windows and Unix conventions `(W/U)`:

```console
<prefix>/(lib/<arch>|lib*|share)/cmake/<name>*/                 (U)  # <-- yours
<prefix>/(lib/<arch>|lib*|share)/<name>*/                       (U)
<prefix>/(lib/<arch>|lib*|share)/<name>*/(cmake|CMake)/         (U)  # <-- mine
```